src/easyscore.ts

```diff
@@ -15,6 +15,7 @@ import { RenderContext } from './types/common';
 import { Accidental } from './accidental';
 import { Modifier } from './modifier';
 import { Voice } from './voice';
+import { Note } from 'note';
```

Comment:
change to `from './note'`

Reply:
While migrating tests, I noticed that I could omit paths for any class under src/.

For example:
```
import { EasyScore } from '../src/easyscore';
import { StaveNote } from '../src/stavenote';
import { System } from '../src/system';
```
can be rewritten as
```
import { EasyScore } from 'easyscore';
import { StaveNote } from 'stavenote';
import { System } from 'system';
```

I think this is due to the baseURL field in the tsconfig.

If VS Code and TS compiler and webpack don't complain, and everything builds fine, does that mean we can omit all the ./ in front of all the imports?

We can do a separate PR for this. I think it would make everything look slightly cleaner if we can omit all the starting ./

Reply:
Yep, agreed.